zerver/views/streams.py

```diff
@@ -371,7 +371,7 @@ def compose_views(thunks: List[Callable[[], HttpResponse]]) -> HttpResponse:
         for thunk in thunks:
             response = thunk()
             if response.status_code != 200:
-                raise JsonableError(response.content)
+                raise JsonableError(response.content)  # nocoverage
```

Comment:
Why does this become nocoverage?

Reply:
Because we used to return `json_error` and then do the error handling using the `status_code` later, which is no longer the case since we now raise `JsonableError` directly.

Reply:
I see, because the middleware runs after this.  I think that means we should delete these two lines, since they're no longer reachable?  (The only way we return a non-200 status code is via throwing an exception).